### PR TITLE
Added SSL netcdf fix for dmci container

### PR DESCRIPTION
**Pull request by magnarem**
**Summary**:

**Related issue**:

**Suggested reviewer(s)**:

**Reviewer checklist**:

* [ ] The headers of all files contain a reference to the repository license
* [ ] 100% test coverage of new code - meaning:
  * [ ] The overall test coverage increased or remained the same as before
  * [ ] Every function is accompanied with a test suite
  * [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  * [ ] Functions with optional arguments have separate tests for all options
* [ ] Examples are supported by doctests
* [ ] All tests are passing
* [ ] All names (e.g., files, classes, functions, variables) are explicit
* [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see [General Conventions](https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html)). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -86,6 +86,9 @@ COPY --from=builder /dst/. /
 
 RUN ls -l /dist/*; for PKG in /dist/*.tar.gz; do pip install $PKG; done
 
+#Fix netcdf4 ssl error
```
Comment (mortenwh) [inline Dockerfile:89]:
```suggestion
# Fix netcdf4 ssl error, occurring when solr-indexer tries to read featureType from the netcdf file
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
kan være greit å få med litt forklaring

File: Dockerfile [hunk 2]
```diff
@@ -86,6 +86,9 @@ COPY --from=builder /dst/. /
 
 RUN ls -l /dist/*; for PKG in /dist/*.tar.gz; do pip install $PKG; done
 
+# Fix netcdf4 ssl error
```
Comment (mortenwh) [inline Dockerfile:89]:
```suggestion
# Fix netcdf4 ssl error, occurring when solr-indexer tries to read featureType from the netcdf file
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
Fikser du kommentaren også?

File: Dockerfile (hunk 2 above)
Comment (magnarem) [reply Dockerfile]: jepp
la til nå

**Review by mortenwh — APPROVED** [review]

**Pull request merged**